docs/Fault-Tolerant-Tachyon-Cluster.md

```diff
@@ -3,12 +3,29 @@ layout: global
 title: Fault Tolerant Tachyon Cluster
 ---
 
+# Fault Tolerance
+
+Fault Tolerance in Tachyon is based upon a multi-master approach where multiple master 
+processes are run.  One of these processes is elected the leader and is used by all workers 
+and clients as the primary point of contact.  The other masters act as standbys using the shared 
+journal to ensure that they maintain the same file system metadata as the leader and can 
+rapidly take over in the event of the leader failing.
+
+If the leader fails a new leader is automatically selected from the available standby masters
+and Tachyon proceeds as usual.  Note that while the switchover to a standby master happens 
+clients may experience brief delays or transient errors.
+
+Note that there is currently no fault tolerance for recovering failed workers.
```

Comment:
There is no real replication (client can cache on multiple nodes) so the act of bringing a new node in the cluster to replace another node doesn't trigger anything; for example replicating the data to the node to reach replication factor.  So the statement that worker does not have a fault tolerance setup is true.